src/IMachineLearningModel.ts

```diff
@@ -0,0 +1,9 @@
+import * as Blockly from 'blockly/core';
+
+interface MachineLearningModel {
+  getSuggestedBlocks(description: string): Array<Blockly.Block>;
+
+  train(data: {description: string; blocks: Blockly.Block[]): void;
```

Comment:
You're missing a closing brace here for the object type annotation here, which is causing the lint error. But remember this should be an Array of objects as you have in: https://github.com/jschanker/suggested-blocks-blockly/blob/a5168f36cba50ad6b5022a93412d3663b0e2912a/IMachineLearningModel.ts#L6